ansible/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: mkdir for config and data
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: "u=rwx,g=rx,o="
+  with_items:
+    - "{{ grafana_config_path }}"
+    - "{{ grafana_lib_path }}"
```

Comment:
Files in /srv/grafana/ are owned by `sshd:crontab` at the moment. Is it possible to allocated separate UID for the service or does `grafana/grafana` need to be started as root?